dot/parachain/types/types.go

```diff
@@ -155,6 +155,39 @@ func (cd CandidateDescriptor) CheckCollatorSignature() error {
 	return sr25519.VerifySignature(cd.Collator[:], cd.Signature[:], payload)
 }
 
+type CandidateDescriptorV2 struct {
+	// The ID of the para this is a candidate for.
+	ParaID ParaID
+	// RelayParent is the hash of the relay-chain block this should be executed in
+	// the context of.
+	RelayParent common.Hash
+	// Version field. The raw value here is not exposed, instead it is used
+	// to determine the `CandidateDescriptorVersion`, see `fn version()`.
+	// For the current version this field is set to `0` and will be incremented
+	// by next versions.
+	Version uint8
+	// The core index where the candidate is backed.
+	CoreIndex uint16
+	// The session index of the candidate relay parent.
+	SessionIndex SessionIndex
+	// Reserved bytes.
+	Reserved1 [25]uint8
```

Comment:
Could we give better naming/descriptions to the two differed reserved bytes fields here?

Reply:
in the RFC they call them as reserved, that's because they want to be compatible with the V1 as they changed some fields, but in their code they just used `reserved` and does not have future plans for them